packages/workbox-webpack-plugin/src/lib/get-webpack-manifest-entries.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+
+const entry = (revision, filePath, publicPath) => ({
+  revision,
```

Comment:
One thing that bears mentioning: as an optimization, the `precache()` function will not append a cache-busting query parameter when populating its caches if the `revision` field is not present.

That's the same behavior that in `sw-precache-webpack-plugin` was defined by the `dontCacheBustUrlsMatching` configuration option, for which we recommended using a RegExp that matches URLs that contained hashes in them already.

I'm not sure exactly how, but is there any indication at this point in the build process whether `filePath` contains a versioning hash in it? (I'd assume it wouldn't for HTML files, but would for most other assets.) If you can safely infer the presence of a hash, then you can leave out `revision` and get optimal behavior without developers needing to use `dontCacheBustUrlsMatching`.

Reply:
Makes sense, we can check if they are using a version hash in the chunk name and filter those out. Then just ignore the `dontCacheBustUrlsMatching` option altogether?

Reply:
If someone goes out of their way to supply `dontCacheBustUrlsMatching` then perhaps it should still be honored, but being able to implicitly provide that optimization is definitely more developer-friendly.

I think it's more of a case of you not taking specific action one way or another in your Webpack-specific code.